packages/datadog-plugin-paperplane/src/index.js

```diff
@@ -36,17 +55,22 @@ const wrapRoutes = tracer => routes => handlers => {
 
 function patch (paperplane, tracer, config) {
   config = web.normalizeConfig(config)
+
+  if (tracer._logInjection) {
+    this.wrap(paperplane, 'logger', wrapLogger(tracer))
+  }
+
   this.wrap(paperplane, 'mount', wrapMount(tracer, config))
   this.wrap(paperplane, 'routes', wrapRoutes(tracer))
 }
 
 function unpatch (paperplane) {
-  this.unwrap(paperplane, ['mount', 'routes'])
+  this.unwrap(paperplane, ['logger', 'mount', 'routes'])
 }
 
 module.exports = {
   name: 'paperplane',
-  versions: ['>=2.3'],
+  versions: ['>=2.3.1'],
```

Comment:
Why the version bump?

Reply:
Because this was needed to make log injection work for errors:
https://github.com/articulate/paperplane/pull/46

Reply:
Support for 2.3.0 could be kept but would just be missing log injection.

Reply:
You know, that's a good point.  Let me see if I can make that work (probably on Monday).  When I do, should I document that difference somewhere?

Reply:
> Let me see if I can make that work

You can find a good example of different feature support by version in the [hapi](https://github.com/DataDog/dd-trace-js/blob/403154a57edcc3168668d38a855f05d23c9d0348/packages/datadog-plugin-hapi/src/index.js#L51) plugin.

> When I do, should I document that difference somewhere?

I would say to add a note in the [docs](https://github.com/DataDog/documentation/blob/master/content/en/tracing/setup/nodejs.md#compatibility), or simply add `paperplane` to the list of supported loggers with `>=2.3.1` as the version.